packages/nodes-base/nodes/Schedule/ScheduleTrigger.node.ts

```diff
@@ -485,10 +511,23 @@ export class ScheduleTrigger implements INodeType {
 			}
 
 			if (interval[i].field === 'hours') {
-				const hour = interval[i].hoursInterval?.toString() as string;
+				const hour = interval[i].hoursInterval as number;
 				const minute = interval[i].triggerAtMinute?.toString() as string;
 				const cronTimes: string[] = [minute, `*/${hour}`, '*', '*', '*'];
 				const cronExpression: string = cronTimes.join(' ');
+				if (hour === 1) {
+					const cronJob = new CronJob(cronExpression, executeTrigger, undefined, true, timezone);
+					cronJobs.push(cronJob);
+				} else {
+					const cronJob = new CronJob(
+						cronExpression,
+						() => executeTrigger(i, hour, 'hours'),
```

Comment:
It's starting to get confusing having these as optional information. We should make this mandatory and change all recurrency rules to send this information and adapt the code to use it whenever needed.